wcivf/apps/elections/templatetags/postcode_tags.py

```diff
@@ -23,3 +23,10 @@ def convert_str_date(value):
 @register.filter(name="totime")
 def convert_str_time(value):
     return datetime.strptime(value, "%H:%M:%S").time()
+
+
+@register.filter(name="anyfuturedates")
```

Comment:
Can you explain why you used a template tag here? I would have thought adding this to the context in the view would have been better, but maybe I'm missing something?

The reason I'm cautious about template tags is that they can quickly cause `n+1` bugs. For example, this tag is assuming that we've called `select_related` on `postelection.election`, and if we've not it could cause loads of queries if not uses correctly.

Reply:
Great point, will make the edit. Thanks.